### PR TITLE
Add @laurens-booij

### PR DESCRIPTION
**Pull request by Laurens-booij**

### CODE REVIEW

File: site/class-2-style/laurens-booij/index.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<meta charset=utf8>
+<title>@laurens-booij</title>
+<link rel=stylesheet href=index.css>
+
+<body></body>
+    <script src=https://d3js.org/d3.v4.min.js></script>
+    <script src=index.js></script>
```
Comment (wooorm) [inline site/class-2-style/laurens-booij/index.html]: `<script>`s outside the body? 😱

**Comment by Laurens-booij** [comment]
Moet ik mijn  hele fork verwijderen? inclusief alle branches

**Comment by wooorm** [comment]
Perfect!

**Comment by wooorm** [comment]
Yup, fo to your fork, then to settings, then remove it, and then fork again!

**Comment by Laurens-booij** [comment]
Okay, thanks!

**Comment by wooorm** [comment]
And after than don’t merge your own PRs, or commit on master. Then you’ll be fine :)

**Pull request merged**